docs/reference/objects/cart/custom_data.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Custom Data
+parent: Cart
+grand_parent: Objects
+---
+
+# Experience slot calendar
+
+Used to display booking custom field values on a [cart object]({% link docs/reference/objects/cart/index.md %}).
+
+## `cart_drop.custom_data`
```

Comment:
This & below can be removed.

Reply:
Still need to remove this, there's no attributes, we just want a little description on line 10. We can update the description to explain it contains the label & value.